pylint/interfaces.py

```diff
@@ -26,16 +26,34 @@
 if TYPE_CHECKING:
     from pylint.reporters.ureports.nodes import Section
 
+__all__ = (
+    "IRawChecker",
+    "IAstroidChecker",
+    "ITokenChecker",
+    "IReporter",
+    "IChecker",
+    "Confidence",
+    "NO_INFERENCE",
+    "HIGH",
+    "INFERENCE",
+    "INFERENCE_FAILURE",
+    "UNDEFINED",
+    "CONFIDENCE_LEVELS",
+)
+
 Confidence = namedtuple("Confidence", ["name", "description"])
 # Warning Certainties
+NO_INFERENCE = Confidence(
+    "NO_INFERENCE", "Warning that is not based on inference result."
+)
 HIGH = Confidence("HIGH", "No false positive possible.")
```

Comment:
What's the difference between `NO_INFERENCE` and `HIGH`?

Reply:
HIGH is the old value I don't want to remove so there is not breaking change, but it's the same. I'm never confident enough to say there won't be any false positive.

Reply:
I would have renamed HIGH but that would be a breaking change.

Reply:
Can't we just rephrase the description of `HIGH` then?

Reply:
You're right.